File: Extras/Tesis Back/prueba2/Models/Licencia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace prueba2.Models
{
    public class Licencia
    {
        public Licencia()
        {
        }

        public int IdLicencia { get; set; }
        public int IdLegajo { get; set; }
        public int Anio { get; set; }
        public int IdTipoLicencia { get; set; }
        public DateTime FecInicio { get; set; }
        public DateTime FecFin { get; set; }
        public String UsrIng { get; set; }
        public String UsrMod { get; set; }
        public String Detalles { get; set; }
        public int CantDiasDisponibles { get; set; }
        public String NombreArchivo { get; set; }
        public String Archivo { get; set; }

    }
}

File: src/app/mis-licencias-solicitadas/modal-mis-licencias-solicitadas/models/Licencia.ts
export class InsertarLicenciaDTO {
  IdLicencia: number | null;
  IdLegajo: number | null;
  IdTipoLicencia: number | null;
  FecInicio: Date | null;
  FecFin: Date | null;
  UsrIng: string | null;
  UsrMod: string | null;
  Detalles: string | null;;
  Anio: number | null;
  CantDiasDisponibles: number | null;
  constructor() { }
}
